packages/meditrak-app-server/src/sync/ServerChangeEnqueuer.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { ChangeHandler } from '@tupaia/database';
+import { MeditrakSyncQueueModel } from '../models';
+import { MeditrakAppServerModelRegistry } from '../types';
+
+type Change = { record_id: string; record_type: string; type: 'update' | 'delete' };
+
+/**
+ * Adds server side changes to the meditrakSyncQueue
+ */
+export class ServerChangeEnqueuer extends ChangeHandler {
```

Comment:
How about that name ey? 😉

It could be better, definitely, but my motivations for going with it are:
1. All our change handlers have verb names describing their job
2. It captures how the actual system operates a bit better than `MeditrakSyncQueue` imo

I felt with the previous name it was a bit misleading, I had assumed that the `meditrakSyncQueue` was responsible both for enqueueing and de-queueing changes. But our system actually has 2 totally separate processes to manage enqueueing changes, and then passing changes to the client. So I like that the name clearly declares that this class just manages one part of the system.

Reply:
Yeah I like the change! Perhaps the `Server` bit could be changed to something like `Syncable` or `Syncing` or something? 
- Not all server side changes are enqueued
- It kinda sounds like something _about_ the server is changing, rather than a database update _on_ the server, but that could just be me and is a minor point

Reply:
I like `SyncableChangeEnqueuer` 👍 definitely better than `Server` I agree!